perception/traffic_light_map_based_detector/src/node.cpp

```diff
@@ -84,6 +55,60 @@ void roundInImageFrame(
   point.y =
     std::max(std::min(point.y, static_cast<double>(static_cast<int>(camera_info.height) - 1)), 0.0);
 }
+
+bool isInDistanceRange(
```

Comment:
Because these functions have nothing to do with the MapBasedDetector, and there have been functions like calcRawImagePointFromPoint3D outside of the class.